airflow/config_templates/config.yml

```
@@ -1300,6 +1300,13 @@
       type: integer
       example: ~
       default: "3"
+    - name: warn_deployment_exposure
+      description: |
+        Boolean for displaying warning for publicly viewable deployment
+      version_added: 2.2.0
```

```suggestion
      version_added: 2.2.3
```

I think that this value will need to be updated, but if this is merged in the next week or two it will probably be included in the 2.2.3 release.

---

Let's make it `2.3.0` please -- this won't be included in 2.2.3

---

```suggestion
      version_added: 2.3.0
```